src/web/components/BusinessContacts/BusinessContact.tsx

```diff
@@ -0,0 +1,57 @@
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+
+import { BusinessContactForm, BusinessContactResponse } from '../../services/participant';
+import BusinessContactDialog from './BusinessContactDialog';
+
+type BusinessContactProps = {
+  contact: BusinessContactResponse;
+  onBusinessContactUpdated: () => void;
+  removeEmailContact: (id: number) => Promise<void>;
+  updateEmailContact: (id: number, form: BusinessContactForm) => Promise<void>;
+};
+
+function BusinessContact({
+  contact,
+  onBusinessContactUpdated,
+  removeEmailContact,
```

Comment:
It's probably more important for these to be consistent because they're the interface to the component - you can rename the functions below as handlers maybe.